binding/HarfBuzzSharp.Shared/DelegateProxies.cs

```diff
@@ -79,7 +76,7 @@ private static void ReleaseDelegateProxyImplementation (IntPtr context)
 		private static IntPtr GetTableDelegateProxyImplementation (IntPtr face, Tag tag, IntPtr context)
 		{
 			var del = GetMulti<GetTableDelegate> (context, out var gch);
-			var blob = del.Invoke (null, tag, null);
+			var blob = del.Invoke (null, tag);
```

Comment:
Do we want to use the UserDataDelegate here as well? If not we should remove the face parameter.

Reply:
Good catch. Let me fix that.